vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib/deps/install-deps.sh

```diff
@@ -71,9 +71,10 @@ if [[ "$platform" == 'darwin' ]]; then
 	xcodebuild -ARCHS="x86_64" -project uv.xcodeproj -configuration Release -target All
 	cp ./build/Release/libuv.a $DEPS_DIR/build/lib
 else
-	./gyp_uv.py -f make
-	make -C out
-	cp ./out/Debug/libuv.a $DEPS_DIR/build/lib
+        sh autogen.sh
+        ./configure --host=aarch64-lmp-linux-gnu
```

Comment:
hardcoding the host arch seems like a bad thing to do.

Reply:
True, should be fixed